src/app/[locale]/layout.tsx

```diff
@@ -0,0 +1,39 @@
+//import React from 'react';
+require("react-dom");
+import Layout from "./components/Layout/Layout";
+import { NextIntlClientProvider } from "next-intl";
+import { locales } from "src/navigation";
+import { getMessages, unstable_setRequestLocale } from "next-intl/server";
+import { Metadata } from "next";
+import { defaultMetadata } from "src/const/seo";
+
+export const metadata: Metadata = defaultMetadata;
+
+type Props = {
+  children: React.ReactNode;
+  params: { locale: string };
+};
+
+export function generateStaticParams() {
+  return locales.map((locale) => ({ locale }));
+}
+
+export default async function LocaleLayout({
+  children,
+  params: { locale },
+}: Props) {
+  unstable_setRequestLocale(locale);
+  // Providing all messages to the client
+  // is the easiest way to get started
```

Comment:
are these comments from the docs?

Reply:
They are. I left them for the next person who contributes. I'll remove them, though.